configs/anchor_free/fcos_rt_r50_fpn_4x.yml

```diff
@@ -0,0 +1,182 @@
+architecture: FCOS
+max_iters: 360000
+use_gpu: true
+snapshot_iter: 20000
+log_smooth_window: 20
+log_iter: 20
+save_dir: output
+pretrain_weights: https://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_cos_pretrained.tar
+metric: COCO
+weights: output/fcos_rt_r50_fpn_4x/model_final
+num_classes: 80
+
+FCOS:
+  backbone: ResNet
+  fpn: FPN
+  fcos_head: FCOSHead
+
+ResNet:
+  norm_type: sync_bn
```

Comment:
freeze_norm is set to True in ResNet and sync_bn is not supported when freeze_norm.
norm_type should change to bn or affine_channel